rust/src/lib.rs

```diff
@@ -145,6 +145,7 @@
 #![recursion_limit="512"]
 
 // create clones of variables that are free to be consumed by a closure
+#[cfg(feature="ffi")]
```

Comment:
This is still needed when not in ffi. It's used when building `'static' closures for functions and maps.

Reply:
Thanks, good catch! Per discussion, I'll change this to `feature="ffi"`.